ios/RNFetchBlob/RNFetchBlob.m

```diff
@@ -42,6 +42,10 @@ + (RCTBridge *)getRCTBridge
     return rootView.bridge;
 }
 
++ (BOOL)requiresMainQueueSetup {
+    return NO;
```

Comment:
We originally forked from https://github.com/wkh237/react-native-fetch-blob, but unfortunately, that repo has been deprecated and archived. The good news is, someone else signed up to be the new maintainer of this repo: https://github.com/joltup/rn-fetch-blob

This small change was based on the new repo: https://github.com/joltup/rn-fetch-blob/blob/master/ios/RNFetchBlob/RNFetchBlob.m#L46

Switching to the newer repo is a much large task, outside the scope of MOB-617. The goal of MOB-617 was mainly to maintain compatibility and keep the Email Integration feature alive.   If we really want to upgrade to the newer repo, and keep the EIv2 feature alive using react-native technology, a new spike ticket should be filed.